CONTRIBUTING.md

```diff
@@ -35,7 +35,8 @@ We welcome many different types of contributions and not all of them need a Pull
 This is a rough outline of what a contributor's workflow looks like:
 
 * Make a fork of the repository within your GitHub account
-* Create a topic branch in your fork from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
+   * Currently we accept contribution from forks only for examples and documentation changes until PR 854 is fixed. Until then please request write priveleges and create a branch in the main repo.
```

Comment:
I had to go and read the PR 854, to understand this statement.
Maybe first the guideline, then the explanation?
`Use forks to only contribute changes on examples and documentation. Currently we accept..`
And emphasise those changes are manually applied, is that correct?

Reply:
OK